### PR TITLE
Embed visualizations

### PR DESCRIPTION
**Pull request by Crashillo**
Closes https://github.com/PopulateTools/issues/issues/1218

## :v: What does this PR do?
Allow embed data visualizations from GobiertoAPI in external sites. [Further reference](https://populate.getoutline.com/doc/gobierto-data-visualizations-embed-PluB5kUNBA)

Just add the following somewhere else:

```html
<div 
  data-gobierto-visualization="<!-- vizz_id -->"
  data-site="<!-- vizz site, i.e. https://datos.gobierto.es -->" 
  data-token="<!-- TOKEN -->" 
></div>

<!-- example -->
<div
  data-gobierto-visualization="158"
  data-site="https://demo-datos.gobify.net"
  data-token="---------------" 
></div>

<script src="https://gobify.net/packs/js/embeds.js"></script>
```

Showcase: https://data-visualizer.populate.tools/public/staging/embed.html

### CODE REVIEW

**Review by ferblape — CHANGES_REQUESTED** [review]
It's not working to me. See where I'm embedding it:

http://downloads.blat.es/test.html

What am I doing wrong? @Crashillo

**Comment by Crashillo** [comment]
Perhaps, it's related to the layout you're implementing. Just for checking, set this style at the body tag: `style="height: 100vh;display:grid;grid-template-columns:1fr 1fr;"` and reload with it

**Comment by Crashillo** [comment]
Yep, definitely, it's a layout issue. I believe I've to set, at least, width & height for the `div` containing the viewer, or even, replace the container itself in order to hide a bit the token

File: app/javascript/gobierto_embeds/index.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    link.media = "screen,print";
+    document.getElementsByTagName("head")[0].appendChild(link);
+  }
+
+  const { data: visualization } = await fetch(`${site}/api/v1/data/visualizations/${id}`, { headers }).then(r => r.json()) || {}
+  if (visualization) {
+    const { attributes: { query_id, sql, spec } } = visualization || {}
+
+    let url = null
+    if (query_id) {
```
Comment (jorgeatgu) [inline app/javascript/gobierto_embeds/index.js]:
```suggestion
const url = query_id ? `${site}/api/v1/data/queries/${query_id}.csv` : sql ? `${site}/api/v1/data/data.csv?sql=${encodeURIComponent(sql)}` : null
```

**Review by jorgeatgu — CHANGES_REQUESTED** [review]

File: app/javascript/gobierto_embeds/index.js (hunk 1 above)
Comment (Crashillo) [reply app/javascript/gobierto_embeds/index.js]: Done

**Pull request merged**